manual_deployment/global-args-security.groovy

```diff
@@ -11,6 +11,7 @@ System.setProperty("hudson.model.ParametersAction.keepUndefinedParameters", "tru
 
 // Variables
 def env = System.getenv()
+def rootURL = env['ROOT_URL']
```

Comment:
1. add "TODO" above this declaration:
//TODO: update with actual values for your CI

2. change default value to null.
It should be executed from script console where no env vars available...

3. jlc.setUrl should be set only when root_url not null
/